src/Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { useCreateSavedAddress } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useCreateSavedAddress"
+import { useUpdateSavedAddress } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useUpdateSavedAddress"
+import { useDeleteSavedAddress } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useDeleteSavedAddress"
+import { useSaveFulfillmentDetails } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useSaveFulfillmentDetails"
+import { useSelectShippingQuote } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useSelectShippingQuote"
+import { useUpdateUserDefaultAddress } from "Apps/Order/Routes/Shipping2/support/ShippingContext/mutations/useUpdateUserDefaultAddress"
+
+export const useShippingMutations = () => {
```

Comment:
a) we don't use index files; b) main named export should be the name of the file, so that its discoverable; c) if we can import mutations from the mutations hook, we don't need to pass it into the context. Just import the hook directly and use that where necessary

Reply:
updated to import the hooks individually. I'd like to make sure they don't re-render since their args don't change but that can wait.